mmcls/datasets/pipelines/auto_augment.py

```diff
@@ -510,18 +510,23 @@ class Solarize(object):
 
     Args:
         thr (int | float): The threshold above which the pixels value will be
-            inverted.
+            inverted when incresing is set to False.
         prob (float): The probability for solarizing therefore should be in
             range [0, 1]. Defaults to 0.5.
+        increasing (bool): When setting to True, the meaning of thr is 
+            8 - actual thr.
     """
 
-    def __init__(self, thr, prob=0.5):
+    def __init__(self, thr, prob=0.5, increasing=False):
         assert isinstance(thr, (int, float)), 'The thr type must '\
             f'be int or float, but got {type(thr)} instead.'
         assert 0 <= prob <= 1.0, 'The prob should be in range [0,1], ' \
             f'got {prob} instead.'
 
-        self.thr = thr
+        if increasing:
+            self.thr = 256 - thr
+        else:
+            self.thr = thr
```

Comment:
I think the `increasing` param should be used in `RandAugment`, and be renamed to `invert`.
Because, in `RandAugment`, it pops the `magnititude_range` key to set the `magnitude` value. If I set `magnititude_range=(0, 128)` for example. Here the actual threshold will become `(128, 256)`, I think that's a confusing behavior.